packages/jsx/src/jsx.d.ts

```diff
@@ -56,51 +57,10 @@ export namespace JSX {
   //   $ServerOnly?: boolean
   // }
   type Accessor<T> = () => T
-  interface Directives {}
```

Comment:
Do not remove these types; they could be used later, and I don't want to sync them with the large Solid sources again.